src/routers/router.ts

```diff
@@ -0,0 +1,34 @@
+import { Router } from 'express';
+import config from '../config';
+import { IGroup, User } from '../group/group.interface';
+import GroupService from '../group/group.service';
+import UserService from '../user/user.service';
+import wrapAsync from '../utils/wrappers';
+
+const router = Router();
+
+// Get group by ID
+router.get('/groups/:id', wrapAsync(async (req, res) => {
+  const groupID = req.params.id;
+  const requesterID = req.headers[config.userHeader] as string;
+
+  const axiosRes = await GroupService.getByID(groupID, requesterID);
+  const group = axiosRes.data as IGroup;
+  const members = group.users;
```

Comment:
A member is not just a userID, it also contains a role.
In the group-service I called it user, but it might be confusing if we also get the user from the userID in the member. Hence, I called it members.